bin/pasta_analyse.py

```diff
@@ -159,69 +159,73 @@ def fill_result(hashes, tag):
         cluster.to_file(f_cluster)
 
     if mbox:
+        if mode == 'succ':
+            log.error('Analysis mode succ is not available in mailbox mode!')
+            return -1
+
```

Comment:
This check is already done before, look at line 146.

Reply:
forgot to remove my check. fixed now